src/lib/operators/operator_join_predicate.cpp

```diff
@@ -70,19 +70,19 @@ OperatorJoinPredicate::OperatorJoinPredicate(const ColumnIDPair& init_column_ids
 void OperatorJoinPredicate::flip() {
   std::swap(column_ids.first, column_ids.second);
   predicate_condition = flip_predicate_condition(predicate_condition);
-  flipped = true;
+  flipped = !flipped;
```

Comment:
😲

Reply:
Someone should add a test for this ;)

Reply:
Oh ... how did that end up here?

Reverting.